src/web/app/components/question-types/question-statistics/contribution-question-statistics/contribution-question-statistics.module.ts

```diff
@@ -0,0 +1,19 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+import { ContributionRatingsListComponent } from './contribution-ratings-list.component';
+import { ContributionComponent } from './contribution.component';
+
+/**
+ * Module for displaying data in a sortable table
```

Comment:
We should describe the function of the module other than where the module is used.

Reply:
Actually I put it under `sortable-table.component.ts` file.

```
/**
 * Displays a sortable table, sorting by clicking on the header
 * Optional sortBy option provided for each column
 * Columns and rows provided must be aligned
 * Remember to register new dynamic components using the withComponents method under sortable-table-module
 */
```

Should I move it or copy it over to the module file?

Reply:
The module itself does not contains any sortable table components so the documentation should not mention it (e.g., consider the case where the module is used in places other than sortable table. It is still suitable to write `Module for displaying data in a sortable table`?).

Reply:
Hmm sorry I'm not too clear about the distinction between Angular components and modules. So what kind of information should I include here?

Reply:
> Now's there's 3 components used by the contribution statistics. Grouping them together would be nice.

Quote your words :P Modules help you organize your component better (there are other benefits like lazy-load which you can see more on the Angular documentation).

Let's put it in this way. When you group similar things together and you want to describe the group. Do you usually describe what group is or where the group is used?

Reply:
> Let's put it in this way. When you group similar things together and you want to describe the group. Do you usually describe what the group is or where the group is used?

Would describe what the group is. But it does gets a bit weird here though, seeing that the module only contains 1 component. 

So i'll just say something like this?

```
/**
 * Sortable table module.
 */
```

Reply:
Ohhh. I think we are talking about two things. The comment I given is for the file: `contribution-question-statistics.module.ts`

Reply:
Ohhh i must have read wrongly. Just changed it.